code/modules/mob/living/carbon/human/say.dm

```diff
@@ -5,6 +5,13 @@
 		verb_ask = pick(T.ask_mod)
 		verb_yell = pick(T.yell_mod)
 		verb_exclaim = pick(T.exclaim_mod)
+	if(wear_mask)
```

Comment:
I think it would be better to hook into this using the signals instead, similarly to how things that affect voice currently do this. (Potentially see megaphone or if not, genetic powers definitely use signals)

Reply:
This doesn't seem to be possible, as with signals you can only modify the message and the spans, but not the displayed name.